tests/test_diskann.cpp

```diff
@@ -329,6 +329,7 @@ TEST_CASE("DiskAnn IP Search", "[ft][diskann]") {
     REQUIRE(recall > 0.70);
 }
 
+/* FIXME: segmentation fault on some platform
```

Comment:
Is it possible to exclude only specific platforms?

Reply:
its hard for now, the crash line is in the BLAS library, maybe we can use another impl of OPQ

Reply:
OK